README.md

```diff
@@ -4,84 +4,171 @@
+dependencies. Move / rename this folder to a convenient location, e.g. to
+`jpscore-deps` as outlined in the beginning.
+
+Additionally you will need to install boost >= 1.74 development packages for
+your distribution.
+
+##### Windows
+
+On Windows dependencies are installed with `vcpkg`, get it from <vcpkg.io>.
+`vcpkg` will automatically download and install the dependencies listen in
+`vcpkg.json` if used with cmake.
+
+During the CMake invocation listed in the next section add this to your invocation:
+```bash
+-DCMAKE_TOOLCHAIN_FILE=<PATH-TO-VCPKG-INSTALLATION>/scripts/buildsystems/vcpkg.cmake
+```
+
+#### How to generate build files with CMake
+
+Now that you have all library dependencies you need to generate build files
+with CMake.
+
+Note: We do not recommend in-tree builds.
 
-#### How to build
-Once you have installed all dependencies and cloned the repository continue
-with a ninja based build:
 ```bash
-mkdir build
-cd build
+mkdir jpscore-build
+cd jpscore-build
 cmake -GNinja -DCMAKE_BUILD_TYPE=Debug <path-to-cmakelists>
 ninja
 ```
 
 Alternatively you can generate a make based build with:
+
 ```bash
-mkdir build
-cd build
+mkdir jpscore-build
+cd jpscore-build
 cmake -DCMAKE_BUILD_TYPE=Debug <path-to-cmakelists>
 make -j$(nproc)
 ```
 
-The following configuration flags are available:
+### Build Options
 
-##### BUILD_DOC defaults to OFF
-Build internal Doxygen based documentation
+For a list of build options please see [CMakeLists.txt] directly.
 
-##### BUILD_CPPUNIT_TEST defaults to OFF
-Build unit tests and add them to ctest
+## Quick start
 
-##### BUILD_TESTING defaults to OFF
-Build full system tests and add them to ctest
+See [Getting started with jupedsim](http://www.jupedsim.org/jpscore_introduction.html).
 
-##### BUILD_WITH_ASAN defaults to OFF (Does not support Windows)
-Build an additional target `jpscore_asan` with address and undefined behavior
-sanitizer enabled. Note there is an approx. 2x slowdown when using
-`jpscore_asan` over `jpscore`
+## Showcase and tutorials
 
-##### CODE_COVERAGE defaults to OFF (Does not support Windows)
-Build unittests with code coverage.
+To highlight some features of JuPedSim we have uploaded some videos on our
+[YouTube channel](https://www.youtube.com/channel/UCKS8w8CUClHEeN4K1SUSMBA).
 
-## Quick start
+## Support
 
-See [Getting started with jupedsim](http://www.jupedsim.org/jpscore_introduction.html).
+If you got a question or a problem and need support from our team feel free to
+contact us. You can do this via [email](mailto:dev@jupedsim.org).
 
-## Showcase and tutorials
+Please do not use the issue tracker for personal support requests.
 
-To highlight some features of JuPedSim we have uploaded some videos on our [YouTube channel](https://www.youtube.com/channel/UCKS8w8CUClHEeN4K1SUSMBA).
+### Reporting bugs and requesting features
 
+If you find a bug in the source code, a mistake in the documentation or think
```

Comment:
bug,

Reply:
I don't think this comma is correct, is it?